docs/getting-started.md

```diff
@@ -97,75 +85,98 @@ components that Molecule provides. These are:
   checking tests (such as deployment smoke tests) on the target
   instance.
 
-## Run test sequence commands
-
-Let's create the first Molecule managed instance with the Docker
-driver.
+The following guide will step through an example of developing and
+testing a new Ansible collection. After reading this guide, you should be
+familiar with the basics of how to use Molecule and what it can offer.
 
-First, ensure that [Docker](https://docs.docker.com/) is running with
-the typical sanity check:
+## Create a collection
 
-```bash
-$ docker run hello-world
-```
+One of the recommended ways to create a collection is to place it
+under the `collections/ansible_collections` directory.
 
-Now, we can tell Molecule to create an instance with:
+To create a collection, use the `ansible-galaxy` tool.
+`ansible-galaxy collection init <namespace>.<collection_name>`
+Example: `ansible-galaxy collection init foo.bar`
```

Comment:
Consider an alternate way to write these sections. Each of these are steps in this procedure of creating a new collection and testing it with molecule.  So for this one, it could be written as:
1. Create a collection.
```
    `ansible-galaxy collection init foo.bar`
```

Reply:
Also notice -this Getting Started is focused on molecule, so we can skip some of the background details about ..well anything that isn't molecule :-) like the naming of collections in this instance.